api/CanMsg.h

```diff
@@ -68,7 +75,10 @@ class CanMsg : public Printable
     size_t len = 0;
```

Comment:
👆did you notice that your copy-assignment operator above is incorrect? I know that isn't part of this PR but I just wanted to point this out.

Reply:
Good catch @thirtytwobits :+1: Thank you :bow: fixed in https://github.com/arduino/ArduinoCore-API/pull/194/commits/e09108034ffb002f366d79646a67607e66fadf45 .